src/core/server/elasticsearch/client/configure_client.ts

```diff
@@ -82,3 +48,36 @@ function ensureString(body: RequestBody): string {
   if ('readable' in body && body.readable && typeof body._read === 'function') return '[stream]';
   return JSON.stringify(body);
 }
+
+function getErrorMessage(error: ApiError, event: RequestEvent): string {
+  if (error instanceof errors.ResponseError) {
+    return `${getResponseMessage(event)} [${event.body?.error?.type}]: ${
+      event.body?.error?.reason ?? error.message
+    }`;
+  }
+  return `[${error.name}]: ${error.message}`;
+}
+
+function getResponseMessage(event: RequestEvent) {
+  const params = event.meta.request.params;
+
+  // definition is wrong, `params.querystring` can be either a string or an object
+  const querystring = convertQueryString(params.querystring);
+  const url = `${params.path}${querystring ? `?${querystring}` : ''}`;
+  const body = params.body ? `\n${ensureString(params.body)}` : '';
+  return `${event.statusCode}\n${params.method} ${url}${body}`;
```

Comment:
It's consistent with the legacy client format, but I wonder if this line break between the status code and the rest of the log message really makes sense.

```
400 GET /_path [undefined]: Response Error
```

seems better than

```
400
GET /_path [undefined]: Response Error
```

Reply:
@pgayvallet I added a comment about the format https://github.com/elastic/kibana/pull/73672/commits/41dad797eea5b2d066eb857990798569bc39ed0b